cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/conformity/modified/CgmesConformity1ModifiedConversionTest.java

```diff
@@ -166,7 +166,7 @@ public void microBEMissingRegulatingControl() {
 
         RatioTapChanger rtc = network.getTwoWindingsTransformer("_b94318f6-6d24-4f56-96b9-df2531ad6543").getRatioTapChanger();
         assertNotNull(rtc);
-        assertFalse(rtc.hasLoadTapChangingCapabilities());
+        assertTrue(rtc.hasLoadTapChangingCapabilities());
```

Comment:
Just for me, could you explain why it this attribute is true now when it was false before? The regulating control is supposed to be missing here and the default value for `loadTapChangingCapabilities` shoud be `false`.

Reply:
Same case as in `microBEInvalidRegulatingControl`.